src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+import UpdateStorage from './Modules/updateStorage.js';
+import { UpdateTask, saveUpdatedTask } from './Modules/edit.js';
+import {
+  displayTasks, deleteTask, add, gameChange, getTodos, demolishAll,
+} from './functions.js';
+
+// variables
+// let list = JSON.parse(localStorage.getItem('storedStTask')) || [];
+
+const taskInject = document.querySelector('.task-injector');
+const taskInput = document.querySelector('.task-input');
+const addBtn = document.querySelector('#add-sign');
+const completeAll = document.querySelector('.complete-delete');
+
+addBtn.addEventListener('click', () => add(taskInput, taskInject));
+taskInput.addEventListener('keypress', (event) => {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    add(taskInput, taskInject);
+  }
+});
+
+// Events
+document.addEventListener('click', (e) => deleteTask(e, taskInject));
+document.addEventListener('change', (e) => gameChange(e));
+completeAll.addEventListener('click', () => demolishAll(taskInject));
+
+taskInject.addEventListener('dblclick', (e) => {
```

Comment:
- [ ] The event listener on taskInject is a little bit complex. It would be more maintainable (KISS principle) and readable if you could break it down into smaller functions, and just call them inside, just as you for the UpdateTask function.
eg: you could create the function inside the event listener on the taskInput outside and call it.